environment.yml

```diff
@@ -17,4 +17,5 @@ dependencies:
     - papermill
     - seaborn
     - ipywidgets
-    - jupysql
\ No newline at end of file
+    - jupysql
+    - jupyter-book
```

Comment:
do we need jupyter-book here? this is the environment for Binder

Reply:
Oops, my mistake